src/main/kotlin/three/consulting/epoc/service/TimesheetService.kt

```diff
@@ -14,6 +14,15 @@ private val logger = KotlinLogging.logger {}
 @Service
 class TimesheetService(private val timesheetRepository: TimesheetRepository) {
 
+    fun findTimesheetForProjectId(id: Long): List<TimesheetDTO>? {
+        logger.info { "Looking for timesheet with project_id: $id" }
+        val timesheet: List<Timesheet> = timesheetRepository.findAllByProjectId(id)
+        if (timesheet.isNotEmpty())
+            return timesheet.map { TimesheetDTO(it) }
+        logger.info { "No timesheet found for id: $id" }
+        return null
```

Comment:
I think this part is a bit redundant. You could just return the empty list instead of null. Nullable lists are something that should be avoided since the empty list does the trick already.

Reply:
And if you want to log whether timesheets were found, you could do something like
```kotlin
logger.info { "Found ${timesheets.size} timesheets for projectId: $projectId}
```

Reply:
Changed to emptyList in 694609c. Also added tests in this commit